src/tools/r2rdump/Amd64/UnwindInfo.cs

```diff
@@ -40,19 +41,24 @@ public enum UnwindFlags
     /// <summary>
     /// based on <a href="https://github.com/dotnet/coreclr/blob/master/src/inc/win64unwind.h">src\inc\win64unwind.h</a> _UNWIND_CODE
     /// </summary>
-    public struct UnwindCode
+    public class UnwindCode
     {
         [XmlAttribute("Index")]
```

Comment:
Do any of these properties need to be set externally from the class? If not, I would suggest using `{get; private set;}`. This lets the property remain public but allows only `UnwindCode` to modify it.

Reply:
When set is private, the XmlSerializer gives an error "Cannot deserialize type 'R2RDump.Amd64.UnwindCode' because it contains property which has no public setter"

Reply:
I hate that thing. Nevermind then!